dbt_copilot_helper/COMMANDS.md

```diff
@@ -23,7 +23,7 @@
 - [copilot-helper copilot make-addons](#copilot-helper-copilot-make-addons)
 - [copilot-helper copilot get-env-secrets](#copilot-helper-copilot-get-env-secrets)
 - [copilot-helper domain](#copilot-helper-domain)
-- [copilot-helper domain check-domain](#copilot-helper-domain-check-domain)
+- [copilot-helper domain configure-domain](#copilot-helper-domain-configure-domain)
```

Comment:
We should rename this command to accommodate our new format/pattern of `copilot-helper noun verb`, so I would suggest something like `copilot-helper domain configure` instead.